src/gui/mainwindow.h

```diff
@@ -100,6 +100,8 @@ class MainWindow : public QMainWindow
     void cleanup();
 
     void showNotificationBaloon(QString title, QString msg) const;
+    // After Application has handled BitTorrent::Session::startupFinished
+    void setEnabledWidgets(bool enabled);
```

Comment:
It's better to declare it private and connect to `Session::startupFinished()` signal somewhere in MainWindow constructor.